src/scss/config/color/_index.scss

```diff
@@ -0,0 +1,32 @@
+@use '~sassdoc-theme-herman/scss/utilities' as herman;
+@use '../tools';
+@use 'sass:meta';
+@use 'brand';
+@use 'background';
+@use 'ccs-defaults';
+@use 'content';
+@forward 'config';
+
+@include herman.add(
+  'colors',
+  'brand-colors',
+  tools.compile-colors(meta.module-variables('brand'))
+);
+
+@include herman.add(
+  'colors',
+  'background-colors',
+  tools.compile-colors(meta.module-variables('background'))
+);
+
+@include herman.add(
+  'colors',
+  'ccs-colors',
+  tools.compile-colors(meta.module-variables('ccs-defaults'))
+);
+
+@include herman.add(
+  'colors',
+  'content-colors',
+  tools.compile-colors(meta.module-variables('content'))
+);
```

Comment:
Most of these aren't showing up in the Herman docs correctly. It looks like before we were only adding `brand-colors` -- should we stick with that? We were also documenting the actual maps before; not sure if that change was intentional.

![Screen Shot 2022-08-09 at 1 49 30 PM](https://user-images.githubusercontent.com/552316/183743770-dc685c41-1159-49e0-a5e0-9994c3ec88c1.jpg)

Reply:
@jgerigmeyer Since Herman doesn't have access to the variables these will all be blank. It is nice to see the value (or variable name) for each color even if we don't see the preview ... whether it is in this form or in the sass map form, but we aren't defining colors in a map anymore, not sure if it makes sense to show the map?

Preferably, we'd see the color in the preview box :)  

Is this possible what I ask near the end of this comment @jgerigmeyer: https://github.com/oddbird/oddleventy/pull/205#pullrequestreview-1057833173

Reply:
We could

1. find a way to render custom-properties in herman color swatches. I think we have some of the infrastructure needed to do that already in sizes?
2. render the map here instead of the color-swatches by creating an 'example' that outputs the result of `meta.module-variables()` (the map that Sass generates for us)

Option 1 is clearly the better solution, if someone has the time to implement it in Herman?

Reply:
Maybe file an issue (or trello card) in Herman, then go with option 2 for now? We could render colors in an iframe with custom CSS like we do for sizes/ratios/icons/examples/fonts.

Reply:
New card created for the longer term fix: https://trello.com/c/6W6pXJtv/93-render-colors-in-herman-in-an-iframe-with-custom-css-like-we-do-for-sizes-ratios-icons-examples-fonts

@mirisuzanne could you help me with 2 above? I'm not sure exactly what this looks like.

Reply:
@stacyk Pushed an example with content-colors